packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.ts

```diff
@@ -40,20 +61,33 @@ export class WizardStepHeaderComponent extends CdkStepHeader implements AfterVie
     /** Whether the given step is selected. */
     @Input() selected: boolean;
 
+    /** Whether the given step is completed. */
+    @Input() completed: boolean;
+
     /** Whether the given step label is active. */
     @Input() active: boolean;
 
     /** Whether the given step is optional. */
     @Input() optional: boolean;
-
     /** Whether the ripple should be disabled. */
     @Input() disableRipple: boolean;
 
     constructor(
         private _focusMonitor: FocusMonitor,
-        _elementRef: ElementRef<HTMLElement>
+        _elementRef: ElementRef<HTMLElement>,
+        @Optional() @Inject(WIZARD_ICONS_PRESET_TOKEN) private injectedIcons: IWizardIcons
     ) {
         super(_elementRef);
+        if (this.injectedIcons) {
+            this.icons = this.getIconsForStates(this.icons, this.injectedIcons);
+        }
+
+    }
+
+    ngOnChanges(changes: SimpleChanges): void {
+        if (changes.customIcons) {
+            this.icons = this.getIconsForStates(this.icons, changes.customIcons.currentValue);
```

Comment:
We don't really need to pass `this.icons` since we can reference `this.icons` in the method implementation itself.